jhrbva/Caravan
# GET endpoint for extracting members of trip

**Related Issue**: [https://caravanning.atlassian.net/browse/CA-87](https://caravanning.atlassian.net/browse/CA-87)

**Description**: Get/members/:tripid
Create GET request to get all the members of a trip 

**Testing PR**: 
You can test this by following these instructions:
1. Run the server ``` node server.js ```
2. Open or Download [Postman](https://www.postman.com/downloads/).
3. To test the functionality create a ``` GET ``` endpoint ``` localhost:8080/members/1 ```
![image](https://user-images.githubusercontent.com/38799293/77034387-3be7d180-6980-11ea-9f48-cf70d5213119.png)
4. Once the above steps are completed, you may click ``` Send ``` (the blue button located towards the top of the application). 
5. This PR is working correctly if you see the following response:
![image](https://user-images.githubusercontent.com/38799293/77034502-8d905c00-6980-11ea-89ad-deea64b42b7e.png)
![image](https://user-images.githubusercontent.com/38799293/77034512-95e89700-6980-11ea-9e0c-16b06a97a762.png)
